e2e-tests/playwright/e2e/plugins/quick-start.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { expect, test } from "@playwright/test";
+import { Common } from "../../utils/common";
+import { UIhelper } from "../../utils/ui-helper";
+
+test.describe("Test Quick Start plugin", () => {
+  let uiHelper: UIhelper;
+  let common: Common;
+
+  test.beforeEach(async ({ page }) => {
+    common = new Common(page);
+    // await common.loginAsKeycloakUser();
+    await common.loginAsGuest();
+
+    uiHelper = new UIhelper(page);
+  });
+
+  test("Access Quick start from Global Header", async ({ page }) => {
+    await uiHelper.clickButtonByLabel("Help");
+    await uiHelper.clickByDataTestId("quickstart-button");
+    await uiHelper.clickButtonByLabel("Help");
+    await uiHelper.clickByDataTestId("quickstart-button");
```

Comment:
@teknaS47 why are these steps repeated twice?

Reply:
Okay, but its not clear what is happening here. To make it more clear, lets first check if the panel is open then use `quickstart-button` to close it and then open it again.